src/app/home/container/categories_list.js

```diff
@@ -0,0 +1,70 @@
+import { connect } from 'react-redux';
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { fetchCategories } from '../../../store/categories/actions';
+import { fetchpostsbyCategory } from '../../../store/posts/actions';
+import CategoriesListItem from '../component/categories_list_item';
+//import { Link } from 'react-router';
+
+class CategoriesList extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this
+      .props
+      .fetchCategories();
+
+    this.onClicked = this
+      .onClicked
+      .bind(this);
+    this.renderCategories = this
+      .renderCategories
+      .bind(this);
+  }
```

Comment:
@jyotigautam the above statements can be in one line IMO @pkrawat1 @zeus999 @kartikjagdale thoughts?